heat/cw4heat/examples/tcw4h.py

```diff
@@ -0,0 +1,40 @@
+from mpi4py import MPI
+
+comm = MPI.COMM_WORLD
+
+import heat.cw4heat as ht
+
+
+with ht.cw4h() as cw:
+    if cw.controller():
+        a = ht.arange(8, split=0)
+        b = ht.ones(8, split=0)
+        c = a @ b
+        # assert hasattr(c, "__partitioned__")
+        print(type(c))
+        p = c.__partitioned__()
+        print(c.shape, c, p)
+        for k, v in p["partitions"].items():
+            print(k, p["get"](v["data"]))
```

Comment:
since we are targeting larger datasets in general, i think that removing the array is enough. If the dict gets so large that its a problem then the `reset` is always an option.